### DISCUSSION TITLE
Boot order frozen despite code change

### DISCUSSION THREAD

**Discussion by freant** [hashicorp/packer-plugin-vsphere#639] [Community Help]
Hello:

I have a problem when I change the boot order in the template's HCL code.

When the VM starts, in the boot sequence in the BIOS, we can no longer modify the sequence, the choices are frozen as if we didn't have the rights.
I can't use the post-process because I don't have anything that could help me with my problem.

**Comment by tenthirtyam** [comment]
Hi there! Unfortunately, more information is needed to assist. 

Please open a bug with as much detail as possible.

Thanks!
